Tasks/Common/nuget-task-common/LocationApi.ts

```diff
@@ -1,7 +1,7 @@
-import Q = require('q');
-import vstsWebApi = require('vso-node-api/WebApi');
-import vstsClientBases = require('vso-node-api/ClientApiBases');
-import VsoBaseInterfaces = require('vso-node-api/interfaces/common/VsoBaseInterfaces')
+import * as Q from "q";
```

Comment:
Same as what it's replacing, just the newer style. Compiles down to the same JS code